### PR TITLE
Add rest api tests for tags

### PR DESCRIPTION
**Pull request by lehinevych**
{{pytest: cfme/tests/test_rest.py -k 'tag' -v --long-running -v}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 4feb3fcde73766cb7c8f5928ed9a7adf855aabaf:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 2451d6a0fdac76921c743a690d99f77d31c8b040:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f0ef1d789e5eeb36884699b1fd175ca17d6ebcfd:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/test_rest.py:1218:25: E261 at least two spaces before inline comment
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 4e882f972d4a809ce85e32fe11a95e50ac49639b:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/test_rest.py:1308:1: W391 blank line at end of file
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d9a4bfc85903732ed5af127f941344cf5be45462:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/test_rest.py [hunk 1]
```diff
@@ -1165,6 +1166,7 @@ def test_import_report(rest_api):
     assert response['message'] == 'Skipping Report (already in DB): [{}]'.format(menu_name)
 
 
+<<<<<<< f2c9f38de81989923db0523866eac4e7bdbed055
```
Comment (jkrocil) [inline cfme/tests/test_rest.py]: This shouldn't be here :)

File: cfme/tests/test_rest.py [hunk 2]
```diff
@@ -1196,6 +1198,109 @@ def test_set_services_owner(rest_api, services):
...
+def test_create_tag_with_wrong_arguments(rest_api):
+    data = {
+        'name': 'test_tag_{}'.format(fauxfactory.gen_alphanumeric().lower()),
+        'description': 'test_tag_{}'.format(fauxfactory.gen_alphanumeric().lower())
+    }
+    try:
+        rest_api.collections.tags.action.create(data)
+    except APIException as e:
+        assert "Category id, href or name needs to be specified" in e.args[0]
+>>>>>>> Add rest api tests for tags
```
Comment (jkrocil) [inline cfme/tests/test_rest.py]: Same here.

**Comment by dajoRH** [comment]
Lint report for commit 659470b316cc56f9b297697cfee901e23315b08f:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/test_rest.py:1169:3: E227 missing whitespace around bitwise or shift operator
./cfme/tests/test_rest.py:1169:3: E901 SyntaxError: invalid syntax
./cfme/tests/test_rest.py:1169:7: E225 missing whitespace around operator
./cfme/tests/test_rest.py:1170:1: E302 expected 2 blank lines, found 0
./cfme/tests/test_rest.py:1201:3: E225 missing whitespace around operator
./cfme/tests/test_rest.py:1201:5: E225 missing whitespace around operator
./cfme/tests/test_rest.py:1201:7: E225 missing whitespace around operator
./cfme/tests/test_rest.py:1202:1: E302 expected 2 blank lines, found 0
./cfme/tests/test_rest.py:1303:7: E225 missing whitespace around operator
```
:trollface:
*CFME QE Bot*

File: cfme/tests/test_rest.py (hunk 1 above)
Comment (lehinevych) [inline cfme/tests/test_rest.py]: you wrote comments faster then I took a look on what I'd added :)

**Comment by dajoRH** [comment]
Lint report for commit 6011013cb13947850b527061947cf6b864cda4d7:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**